### PR TITLE
send video type to rtcstats

### PR DESCRIPTION
**Pull request by gpolitis**
With this PR meet will send the video type (screensharing or camera) of remote video tracks.

### CODE REVIEW

File: react/features/rtcstats/middleware.js [hunk 1]
```diff
@@ -52,6 +53,39 @@ MiddlewareRegistry.register(store => next => action => {
...
+                    videoType
+                });
+            }
+        }
+        break;
+    }
+    case TRACK_UPDATED: {
+        if (canSendRtcstatsData(state)) {
+            const videoType = action.track?.videoType;
+            const jitsiTrack = action.track.jitsiTrack;
```
Comment (damencho) [inline react/features/rtcstats/middleware.js:74]: Why skipping the undefined check here, when you have it above?

File: react/features/rtcstats/middleware.js [hunk 2]
```diff
@@ -52,6 +53,39 @@ MiddlewareRegistry.register(store => next => action => {
         }
         break;
     }
+    case TRACK_ADDED: {
+        if (canSendRtcstatsData(state)) {
+            const jitsiTrack = action.track.jitsiTrack;
```
Comment (damencho) [inline react/features/rtcstats/middleware.js:58]: I guess when adding tracks action.track cannot be undefined :)

**Review by damencho — CHANGES_REQUESTED** [review]
A small change, other than that LGTM.

File: react/features/rtcstats/middleware.js (hunk 2 above)
Comment (gpolitis) [reply react/features/rtcstats/middleware.js:58]: Right, but I am always worried about these "impossible" exceptions happening at random situations and places.. That said, we should be good here.

Comment (gpolitis) [reply react/features/rtcstats/middleware.js:58]: We should be good, but I will make the code more defensive, just in case!

**Review by damencho — APPROVED** [review]

**Pull request merged**